### PR TITLE
Added svgo-loader for cleaning SVG

### LINKED ISSUE #507: icons svg are not cleaned correctly, there is still some doctype in the page o_O

### PR DESCRIPTION
**Pull request by magsout**
Close #507

### CODE REVIEW

**Comment by MoOx** [comment]
This is not enough from what I know. the SVGIcon component should cleanup things that svgo does not, but I guess there is still a bug in SVGIcon :/
I heard several times that svgo can be buggy and sometimes alter svg. Not sure about that point tho.

**Comment by MoOx** [comment]
Ok maybe my previous comment was shit. Is svgo removing the doctype? If so, looks good to me.

File: scripts/build.js [hunk 1]
```diff
@@ -115,7 +121,10 @@ const webpackConfig = {
       },
       {
         test: /\.(svg)$/,
-        loader: "raw-loader",
+        loaders : [
+          "raw",
```
Comment (MoOx) [inline scripts/build.js]: Maybe hard code "-loader", it's more clear imo

**Comment by MoOx** [comment]
If nobody is not happy, let's merge.

**Comment by MoOx** [comment]
@magsout note: it's `PR: review *`, not `Post: review *` for this kind of pr :p

**Comment by magsout** [comment]
>Ok maybe my previous comment was shit. Is svgo removing the doctype? If so, looks good to me.

yes of course.


Ah ah my bad about label... 😅

File: scripts/build.js (hunk 1 above)
Comment (magsout) [inline scripts/build.js]: I remove loader because we can add only loader in `loaders`

**Pull request merged**